paging/guava/src/main/java/androidx/paging/ListenableFutureRemoteMediator.kt

```diff
@@ -75,7 +75,7 @@ abstract class ListenableFutureRemoteMediator<Key : Any, Value : Any> :
      *  refresh request from the UI before dispatching a [load] with load type [LoadType.REFRESH].
      */
     open fun initializeFuture(): ListenableFuture<InitializeAction> {
-        return ResolvableFuture.create<InitializeAction>()
+        return SettableFuture.create<InitializeAction>()
```

Comment:
`Futures.immediateFuture(LAUNCH_INITIAL_REFRESH)` is a bit more idiomatic here

Reply:
We tend to avoid `SettableFuture` outside of tests because it has a few weird quirks (although they are not applicable here), still since we bring in all of guava now we can just use `Futures.immediateFuture` directly.

Reply:
Fixed! Good to know, my Guava knowledge is quite limited 😅